stylometryproject/stylometryapp/static/scripts/docDisplay.js

```diff
@@ -88,29 +94,34 @@ $('#profile-files-list').on('click', '.delete-document-button', function (event)
     var csrftoken = $('input[name=csrfmiddlewaretoken]').val();
     console.log(csrftoken);
 
-    var documentId = $(this).data('documentId');
+    var documentId = $(this).data('documentid');
```

Comment:
Is the field in .data supposed to be "data-documentID" since that is what you set for the cloned delete button on line 35?

Reply:
yes it is